### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#1402] [Discussão]
-

**Comment by HmmerHead** [comment]
> **1. Como podemos demonstrar nossas melhores qualidades para as pessoas?**

Se pensarmos em um nível técnico, ter um conteúdo produzido, e de fácil acesso ajuda nisso. Seja por aqui, ou com algo que esteja no ar por ex.
Em um nível social, seria ir em eventos, e ser visto pela comunidade, para que outros saibam mais sobre seus jeito de ser e interagir. 

> **2. Como podemos mostrar que o Brasil tem vários front-enders bons para o mundo? Quais você considera importantes? Quais são suas referências?**

Acredito que para essa questão, uma das muitas possibilidades, seria no desenvolvimento de novas linguagens, ou ferramentas, ou ainda no apoio das já existentes. Penso inclusive, que ter na equipe, indivíduos de várias nacionalidades, fortalece na pluralidade de conhecimento, haja vista que cada um vive em uma realidade diferente, e experienciou algo diferente, que poderá agregar no todo. 

> **5. Como você ajuda as pessoas?**

Quando sei o assunto, tento ajudar o máximo possível.


_________
_Falei sobre alguns pontos, mas para aqueles que tem mais experiencia, façam a sua menção a outros detalhes importantes_

**Comment by guilhermealveslopes** [comment]
> **1. Como podemos demonstrar nossas melhores qualidades para as pessoas?**

Nessa pergunta da pra fazer um link com a 5ª.

Eu sempre fiz meu network simplesmente sendo sincero, esforçado e motivado pra fazer tudo que me foi proposto ao longo dos anos.
Nunca precisei falar que eu sei, eu sempre preferi ajudar as pessoas e fazê-las entenderem que eu tenho qualidades, que bem aproveitadas, ambos serão beneficiados.
Trabalhar duro, sem se preocupar com a grana, e sim com as pessoas, faz você mudar o jogo quando alguém está avaliando suas qualidades.

> **3. Como conseguir clientes?** 

São todas as perguntas meio lincadas.
Conseguir clientes é mais fácil do que parece, aquele que não ta conseguindo, é por que está tentando pular alguma etapa.
Não adianta querer fechar um job de R$10k - R$20k
se você nunca fez um de R$1k, não ajudou ninguém da comunidade, não fez projetos pessoais pra intensificar seus conhecimentos.
Pra mim conseguir clientes se baseia unica e exclusivamente de fazer seu trabalho bem feito.
Quando eu comecei a agir assim, foi quando comecei a escolher os projetos em que ia trabalhar, por que o mercado realmente ta CHEIO de trabalho.

> **5. Como você ajuda as pessoas?**

Assim como já foi dito, eu não gosto de entregar as respostar de mão beijada.
E não gosto quando alguém que me importo está escolhendo o lado 'gambiarra' da força.
Não tem nada de errado nisso, mas os bons costumes na hora de programar são importantes, e fazer as pessoas entenderem todo o contexto do problema, geralmente é a minha tática.
Acredito que os melhores networkings são feitos quando você ajuda pessoas sem 2ª intenções.
Quando eu ajudo, é por que sou fascinado em resolver problemas, a minha recompensa é ver o próximo feliz :)
